samples/java/confluent/src/main/java/com/contoso/kafka/KafkaTriggerRetry.java

```diff
@@ -13,36 +13,16 @@ public class KafkaTriggerRetry {
     @FunctionName("KafkaTriggerRetry")
     @FixedDelayRetry(maxRetryCount = 3, delayInterval = "00:00:02")
     public void runSingle(
-            @KafkaTrigger(
-                    name = "KafkaTrigger",
-                    topic = "topic",
-                    brokerList="%BrokerList%",
-                    consumerGroup="$Default",
-                    username = "$ConnectionString",
-                    password = "EventHubConnectionString",
-                    authenticationMode = BrokerAuthenticationMode.PLAIN,
-                    protocol = BrokerProtocol.SASLSSL,
-                    dataType = "string"
-            ) String kafkaEventData,
+            @KafkaTrigger(name = "KafkaTrigger", topic = "topic", brokerList = "%BrokerList%", consumerGroup = "$Default", username = "$ConnectionString", password = "EventHubConnectionString", authenticationMode = BrokerAuthenticationMode.PLAIN, protocol = BrokerProtocol.SASLSSL, dataType = "string") String kafkaEventData,
```

Comment:
Why are we using EventHubConnectionString in confluent sample?

Reply:
resolved